CRM/Contact/BAO/Query.php

```diff
@@ -2374,7 +2374,7 @@ public static function getLocationTableName(&$where, &$locType) {
       list($tbName, $fldName) = explode(".", $where);
 
       //get the location name
-      $locationType = CRM_Core_PseudoConstant::get('CRM_Core_DAO_Address', 'location_type_id');
+      $locationType = CRM_Core_PseudoConstant::get('CRM_Core_DAO_Address', 'location_type_id', ['labelColumn' => 'name']);
```

Comment:
So the comments on that function are

```
   * Low-level option getter, rarely accessed directly.
   * NOTE: Rather than calling this function directly use CRM_*_BAO_*::buildOptions()
   * @see http://wiki.civicrm.org/confluence/display/CRMDOC/Pseudoconstant+%28option+list%29+Reference
```

I think getName makes more sense where we can - or buildOptions if we really need to get all - has a few different contexts